package.json

```diff
@@ -29,15 +29,15 @@
   },
   "dependencies": {
     "@fastify/csrf": "^5.0.0",
-    "fastify-plugin": "^3.0.0",
-    "http-errors": "^2.0.0"
+    "@fastify/error": "^3.0.0",
+    "fastify-plugin": "^3.0.0"
   },
   "devDependencies": {
-    "@types/node": "^18.0.0",
-    "fastify": "^4.0.0-rc.2",
     "@fastify/cookie": "^7.0.0",
     "@fastify/secure-session": "^5.0.0",
     "@fastify/session": "^9.0.0",
+    "@types/node": "^18.0.0",
+    "fastify": "^4.0.0-rc.2",
```

Comment:
why not final version?

Reply:
npm i reordered the entries. Any suggestion?

Reply:
let's bump to latest while we are at it :(

Reply:
```suggestion
    "fastify": "^4.2.0",
```